### DISCUSSION TITLE
Is /dev/fuse useable in github actions?

### DISCUSSION THREAD

**Discussion by bergwolf** [community/community#26404] [Actions]
<p>Hello,</p>
<p>I have a project that need <code>/dev/fuse</code> device to test. Currently its CI is running on travis and I tried to migrate to github actions but it seems that <code>/dev/fuse</code> is not accessible as I’m getting EINVAL when reading from it although I can open it w/o issues.</p>
<p>Is there any extra config to enable access to <code>/dev/fuse</code> device?</p>

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/bergwolf">@bergwolf</a>,</p>
<p>The FUSE seems is not pre-installed on the GitHub-hosted runners.<br>
You can try to use the following ways to install FUSE on the GitHub-hosted runners:</p>
<ul>
<li>
<p><strong>On Linux runner</strong></p>
<pre><code class="lang-bash">sudo apt-get update
sudo apt-get install fuse
</code></pre>
</li>
<li>
<p><strong>On macOS runner</strong></p>
<pre><code class="lang-bash">brew cask install osxfuse
</code></pre>
</li>
</ul>
